### DISCUSSION TITLE
How can I watch my pics in code-server

### DISCUSSION THREAD

**Discussion by Sdamuu** [coder/code-server#6097] [Q&A]
as title. I can't watch my pics in folder, how can I solve it? 
Is there any tools or plugine can solve it?
Install Environment: Centos 7.6, Version 4.11.0, the lastest one.

**Comment by code-asher** [comment]
You should be able to view images.  Is there a specific file type that is not working?

**Reply by Sdamuu** [reply]
just a normal JPG image, when i click it, show me like this:
![test_pic](https://user-images.githubusercontent.com/23159675/227125239-8f855a02-d242-4286-8696-0856dc12c9a9.png)

**Reply by code-asher** [reply]
Interesting, that should work.  I tried with 4.11.0 and it worked for me.  Are there any logs in the browser console?

**Reply by Sdamuu** [reply]
nothing, both operation log and console....😂😂

**Reply by code-asher** [reply]
No logs at all?

Code opens images in an iframe so it should just work but maybe the iframe is being blocked for some reason (although there would be a warning in the browser logs if this is the case).

**Comment by rourke** [comment]
Can it be an permission issue? Is it readable for the code-server user?

**Reply by Sdamuu** [reply]
yes, it's a readable image file, in same folder, all files except image files are ok...
